### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by shadiakiki1986** [thulasi-ram/ivory#1]
Hello. Is this project unmaintained? How does it compare to https://github.com/MicroPyramid/Django-CRM ?

**Comment by thulasi-ram** [comment]
@shadiakiki1986 

### TLDR: If you are looking for something to quickly implement and make it to production go with MicroPyramid CRM. If you are not in a hurry you can watch this repo. If you are looking for contributing you are more than welcome.

Ivory is not a complete project yet. I am working on this as I plan to implement this as a business offering. But it might take a while. MicroPyramid's CRM is more feature complete. Ivory takes a different position towards scalability. I believe scalability is baked into Ivory. I have thought (maybe over thought as well) at every step of ivory.

**Differences:**
- Their domain structure is a bit different from Ivory. Ivory supports GST. Their current structure does nothing to prevent duplicate data once someone starts entering data. This is the single most troubling issue when dealt with in scale.
- They are heavily dependent on models and forms which I am not a fan of these things. IMO they start hurting productivity at scale again.
- MicroPyramid CRM is also mostly dependent on Django served pages. In my experience, this again doesn't go well with scale. Ivory will be delivering the front end with Vue.js in the near future.

I have designed a CRM in the past and have faced all the issues stated above once the clientele has become more than 10k. The code base grew from only me to a 10 member team and I have an idea how the code base gets worse. How the data gets polluted if it is entered by people who were hired to just enter data. How the pages start becoming slow, how the product requirement changes in a way that messes up the code base and I can see all of them in MicroPyramid CRM. Although they did a wonderful job with the CRM I am not a fan of their style.

**Comment by thulasi-ram** [comment]
@shadiakiki1986 Was I able to address your query?

**Comment by shadiakiki1986** [comment]
Yes absolutely! Thanks and best of luck

**Comment by thulasi-ram** [comment]
Thanks a lot.
